kratos/python/add_utilities_to_python.cpp

```diff
@@ -524,13 +524,31 @@ void AddUtilitiesToPython(pybind11::module& m)
         .def("UpdateSearchDatabase", &BinBasedFastPointLocator < 2 > ::UpdateSearchDatabase)
         .def("UpdateSearchDatabaseAssignedSize", &BinBasedFastPointLocator < 2 > ::UpdateSearchDatabaseAssignedSize)
         .def("FindPointOnMesh", &BinBasedFastPointLocator < 2 > ::FindPointOnMeshSimplified)
+        .def("FindPointOnMesh", [](BinBasedFastPointLocator < 2 >& self, const array_1d<double,3>& coords, const double tol)
+            {
+                Element::Pointer pelem;
+                Vector N;
+                int max_results = 1000;
```

Comment:
I would not put 1000 as a fix number 
I would try to mimic  a dynamic array starting from size and then reducing it to number of correct results

Reply:
the user does not see this. It is required by the search function ... so it is actually out of my control

Reply:
So you should allocate as the size of the objects in search. If there are more than 1000, then it would give 1000s of them which may or not include the nearest ones.....

Reply:
the parameter 1000 is used internally already (in the c++ counterpart). I would only let the user select the tolerance, but this cannot be done because in c++ one cannot prescribe just one of the default parameters